### PR TITLE
Add functionality to main window UI 

### PR DESCRIPTION
**Pull request by ayaNader**
This PR adds some functionality to parts of the extension's main window such as events for the About and Help buttons which cause corresponding windows to be opened. As well as adding the extension version of tracking list of readers.

### CODE REVIEW

File: smart_card_connector_app/src/window-about-showing.js [hunk 1]
```diff
@@ -33,6 +34,8 @@ const ABOUT_WINDOW_OPTIONS = {
   'id': 'about-window',
   'innerBounds': {'width': 700, 'height': 500}
```
Comment (emaxx-google) [inline smart_card_connector_app/src/window-about-showing.js:35]: nit: Let's use the new constants here as well.

**Review by emaxx-google — CHANGES_REQUESTED** [review]

File: smart_card_connector_app/src/window-about-showing.js [hunk 2]
```diff
@@ -46,7 +51,16 @@ const openAboutElement =
 function openAboutClickListener(e) {
   e.preventDefault();
 
-  chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  if (GSC.Packaging.MODE === GSC.Packaging.Mode.APP) {
+    chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
```
Comment (emaxx-google) [inline smart_card_connector_app/src/window-about-showing.js:55]: nit: Actually, let's maybe delete the ABOUT_WINDOW_OPTIONS constant and just inline it here? It's a weird asymmetry that we have a constant for the App-specific API but no constant or the Extension-specific API.

**Review by emaxx-google — APPROVED** [review]
LGTM % nit

File: smart_card_connector_app/src/window-about-showing.js (hunk 2 above)
Comment (ayaNader) [reply smart_card_connector_app/src/window-about-showing.js:55]: wouldn't inlining it be too crowded?

Comment (emaxx-google) [reply smart_card_connector_app/src/window-about-showing.js:55]: A matter of taste I guess, but for this case - where the constant is only useful for one conditional branch but not for the other - I think it's fine.

If you don't like it, then what about doing it the other way round, i.e., extracting the chrome.windows.create's argument into a global constant? And clearing naming one as a app-specific and another as an extension-specific constant?

**Review by emaxx-google — APPROVED** [review]
LGTM!

**Pull request merged**